crates/swc_ecma_minifier/tests/terser/compress/comparing/lit_comparisons/config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please move this to `tests/compress/fixture/issues/**`
`terser` directory is directory for tests ported from terser.

Reply:
Ok, moved.